CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+## v1.0.2- Feature: Added formatting option

```

Comment:
You add a feature, hence increase the minor version -- not the patch version.

Reply:
Done.